deepfence_server/model/registry.go

```diff
@@ -622,33 +663,27 @@ func RegistrySummary(ctx context.Context, registryID mo.Option[string], registry
 	defer tx.Close()
 
 	queryPerRegistry := `
-	MATCH (n:RegistryAccount{node_id:$id})-[:HOSTS]->(m:ContainerImage)-[:IS]-(i:ImageStub)
-	WITH n,
-		COUNT(distinct m.docker_image_name) AS images,
-		COLLECT(distinct i.tags) AS tags,
-		COUNT(distinct n) AS registries
+    MATCH (n:RegistryAccount{node_id:$id})
+    WITH COUNT(distinct n) as registries
+	MATCH (n:RegistryAccount{node_id:$id})-[:HOSTS]->(i:ImageStub)-[:IS]-(m:ContainerImage)
+	MATCH (n)-[:HOSTS]->(m)
+	WITH registries, n,
+		COUNT(distinct i.docker_image_name) AS repositories,
+		COUNT(distinct m.docker_image_id) AS images
```

Comment:
Same here, let's make a split to retrieve the `count` of registries
Makes little sense since it's just duplicated

Reply:
Done.